example/app/AppSync.tsx

```diff
@@ -1,4 +1,4 @@
-import React, {useCallback, useEffect, useMemo} from 'react';
+import React, {useCallback, useEffect} from 'react';
```

Comment:
Only if you want to omit `React.` on line 18.
```suggestion
import React, {useCallback, useEffect, useState} from 'react';
```